docs/spec/literal.rst

```diff
@@ -584,36 +581,36 @@ Type inference
 Inferring ``LiteralString``
 """""""""""""""""""""""""""
 
-Any literal string type is compatible with ``LiteralString``. For
+Any literal string type is assignable to ``LiteralString``. For
 example, ``x: LiteralString = "foo"`` is valid because ``"foo"`` is
 inferred to be of type ``Literal["foo"]``.
 
 We also infer ``LiteralString`` in the
 following cases:
 
 + Addition: ``x + y`` is of type ``LiteralString`` if both ``x`` and
-  ``y`` are compatible with ``LiteralString``.
+  ``y`` are assignable to ``LiteralString``.
 
 + Joining: ``sep.join(xs)`` is of type ``LiteralString`` if ``sep``'s
-  type is compatible with ``LiteralString`` and ``xs``'s type is
-  compatible with ``Iterable[LiteralString]``.
+  type is assignable to ``LiteralString`` and ``xs``'s type is
+  assignable to ``Iterable[LiteralString]``.
 
 + In-place addition: If ``s`` has type ``LiteralString`` and ``x`` has
-  type compatible with ``LiteralString``, then ``s += x`` preserves
+  type assignable to ``LiteralString``, then ``s += x`` preserves
```

Comment:
Feels like there's a missing word here. "`x` has a type assignable to" would probably be correct.